frameworks/Java/play2-java/play2-java-ebean-hikaricp/conf/application.conf

```diff
@@ -19,6 +19,9 @@ play.server {
   netty {
     transport = "native"
 
+    # Whether the Netty wire should be logged
+    log.wire = false
+
```

Comment:
Added this to all `application.conf` files. Disabling a logging setting, as it's not needed. And then 2 settings which, according to the documentation, should have better performance.